### DISCUSSION TITLE
How can I get the data in the tests earth files?

### DISCUSSION THREAD

**Discussion by geobeans** [pelicanmapping/osgearth#1844] [Support Q&A] [answered]
In the constraints.earth:
   <OGRFeatures name="data:ridgelines">
        <url>D:/data/demo/nepal/nepal_ridgetops_50m.shp</url>
    </OGRFeatures>

    <OGRFeatures name="data:airports">
        <url>D:/data/airports/APTBoundary_all.shp</url>
    </OGRFeatures>

    <OGRFeatures name="data:coastlines">
        <url>D:/data/noaa/tests.shp</url>
    </OGRFeatures>

   How can I get the shp files in the earth file?

**Comment by gwaldron** [comment, marked as answer]
This is all data we cannot distribute directly, but you can find the data elsewhere:
The airport boundary shapefiles are originally from X-Plane.
Look here for coastline vector data: https://osmdata.openstreetmap.de/data/coastlines.html
